shared/src/clients/phonebook-client.class/index.ts

```diff
@@ -59,4 +60,21 @@ export default class PhonebookClient {
       throw new Error(err as any)
     }
   }
+
+  public async getUniqueLinksForUsers(
+    body: PhonebookChannelDto
+  ): Promise<UserChannel[]> {
+    try {
+      const res = await this.request(
```

Comment:
Lets continue the practice of circuit breaker. We dont want campaigns to halt or postman to slowdown if phonebook isnt responding